server/speak.py

```diff
@@ -1,12 +1,13 @@
 # speak.py
 from gtts import gTTS
-from playsound import playsound
-
-FILENAME = "tts.mp3"
+import os
+import filename
+import delete_function
 
 def get_speak (text, lang):
-    print("speak" + text)
-    print("speaklang" + lang)
+    FILENAME = filename.random_filename()
     tts = gTTS(text=text, lang=lang, slow=False)
     tts.save(FILENAME)
-    playsound(FILENAME)
\ No newline at end of file
+    delete_function.delete_all()
+
+mp3_path = os.path.join(os.path.dirname(__file__), "../static/dist/FILENAME")
```

Comment:
- `get_speak()` should return the new random file we've created
- `delete_all()` should be executed before `tts.save()`. what if you deleted the file we've just created ?
- `FILENAME` we usually capitalize global variables, to improve code readability. in this case it's a local variable so that's not needed.

**Example**:
```python
filename = filename.random_filename()

delete_function.delete_all()
tts = gTTS(text=text, lang=lang, slow=False)
tts.save(filename)
return filename
```

Reply:
Good corrections. Thanks !